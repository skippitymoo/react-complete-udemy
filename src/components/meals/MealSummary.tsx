import { FC } from 'react';

export const MealSummary: FC = () => {
  return (
    <section className='meal-summary'>
      <h2 className='meal-summary__title'>Delicious Food, Delivered To You</h2>
      <p className='meal-summary__text'>
        Choose your favourite meal from our broad selection of available meals and enjoy a delicious
        lunch or dinner at home.
      </p>
      <p className='meal-summary__text'>
        All our meals are cooked with high-quality ingredients, just-in-time and of course by
        experienced chefs!
      </p>
    </section>
  );
};
