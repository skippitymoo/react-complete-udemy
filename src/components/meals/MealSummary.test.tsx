import { render } from '@testing-library/react';
import { MealSummary } from './MealSummary';
import { axe, toHaveNoViolations } from 'jest-axe';

expect.extend(toHaveNoViolations);

describe('MealSummary', () => {
  it('should pass axe', async () => {
    const { container } = render(<MealSummary />);
    expect(await axe(container)).toHaveNoViolations();
  });

  it('should render as expected', async () => {
    const { container } = render(<MealSummary />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
