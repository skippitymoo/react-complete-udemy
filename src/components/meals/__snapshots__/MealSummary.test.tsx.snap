// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MealSummary should render as expected 1`] = `
<section
  class="meal-summary"
>
  <h2
    class="meal-summary__title"
  >
    Delicious Food, Delivered To You
  </h2>
  <p
    class="meal-summary__text"
  >
    Choose your favourite meal from our broad selection of available meals and enjoy a delicious lunch or dinner at home.
  </p>
  <p
    class="meal-summary__text"
  >
    All our meals are cooked with high-quality ingredients, just-in-time and of course by experienced chefs!
  </p>
</section>
`;
